### PR TITLE
Feature 80 scripting api

### LINKED ISSUE #80: Scripting api
As an story developer I would like to have possibility to run javascript files to perform different tasks

### PR DESCRIPTION
**Pull request by hkurhinen**
fixes #80

### CODE REVIEW

**Comment by hkurhinen** [comment]
DB migrations missing. Do not merge yet.

File: src/main/java/fi/metatavu/metamind/bot/functions/RunScriptMetaBotFunction.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        logger.error(String.format("Module with name %s not found", moduleName));
+        return;
+      }
+
+      Value module = modules.getMember(moduleName);
+      if (!module.hasMember(functionName) || module.getMember(functionName).canExecute()) {
+        logger.error(String.format("Function with name %s does not exits or is not executable", functionName));
+      }
+
+      Value returnValue = module.getMember(functionName).execute();
```
Comment (hkurhinen) [inline src/main/java/fi/metatavu/metamind/bot/functions/RunScriptMetaBotFunction.java]: Should we provide possibility to pass parameters to script function?

Comment (anttileppa) [reply src/main/java/fi/metatavu/metamind/bot/functions/RunScriptMetaBotFunction.java]: yes

Comment (hkurhinen) [reply src/main/java/fi/metatavu/metamind/bot/functions/RunScriptMetaBotFunction.java]: Added support for parameters

**Pull request merged**